.cargo/config.toml

```diff
@@ -0,0 +1,10 @@
+[target.x86_64-pc-windows-msvc]
+# Increase default stack size to match Linux
```

Comment:
Not a fan of this comment.

We don't increase it to match Linux, but just because we need it for Slint in debug mode.
Also the default stack size on Linux is 8MiB AFAIK.